plugins/policy/cache/cache_impl.go

```diff
@@ -240,8 +245,9 @@ func (pc *PolicyCache) LookupNamespace(namespace nsmodel.ID) (found bool, data *
 // LookupNamespacesByLabelSelector evaluates label selector (expression
 // and/or match labels) and returns IDs of matching namespaces.
 func (pc *PolicyCache) LookupNamespacesByLabelSelector(
-	nsLabelSelector string) []string {
-	return pc.configuredNamespaces.LookupNamespacesByLabelSelector(nsLabelSelector)
+	nsLabelSelector string) []nsmodel.ID {
+	namespaces := pc.configuredNamespaces.LookupNamespacesByLabelSelector(nsLabelSelector)
+	return utils.UnstringNamespaceID(namespaces)
 }
```

Comment:
A little nit:  the name of the function should be utils. UnstringNamespaceIDs (with and 's' at the end to indicate plural)